client/reader/stream/index.jsx

```diff
@@ -459,8 +461,10 @@ module.exports = React.createClass( {
 			showingStream = true;
 		}
 
+		const StreamMain = config.isEnabled( 'reader/refresh/stream' ) ? ReaderMain : Main;
```

Comment:
usually we want to put each item in the ternary on a new line unless it's very small. this case may fit but I think it could be a bit better on three lines…

```js
const StreamMain = config.isEnabled( 'reader/refresh/stream' )
	? ReaderMain
	: Main;
```